### PR TITLE
[DE] Add `History of osu!/2022`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
Translating self-contributed articles :muscle:

- [x] Find translation for `rate adjustments` (line 19)

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/History_of_osu!/2022/de.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+osu!(lazer) erhielt die Mod Freeze Frame, die jeweils nur eine Combo anzeigt, was mehr Lesevermögen von den Spielern verlangt.[^freeze-frame]
+
+Ein Update auf der osu!-Webseite machte es möglich, den höchsten Rang eines Spielers, den er jemals erreicht hat, anzuzeigen, sobald man mit der Maus über den Spielerrang auf dem Profil fuhr.[^highest-rank]
+
+Für das diesjährige Halloween gab es erneut einen [Fanart-Wettbewerb](https://osu.ppy.sh/home/news/2022-10-10-halloween-fanart-contest), bei dem Künstler dazu aufgefordert wurden, die osu!-Maskottchen in gespenstischen Kostümen zu zeichnen. [Dreamxiety](https://osu.ppy.sh/users/13103233) gewann den ersten Platz im Wettbewerb mit 3.749 Stimmen.[^halloween] Eine Einsendung wurde widerrufen, weil sie als zu angsteinflösend angesehen wurde und dem Autor wurde stattdessen ein eigenes Abzeichen verliehen.[^grin-reddit][^grin-twitter][^grin-badge]
```
Comment (The-Last-Cookie) [inline wiki/History_of_osu!/2022/de.md:113]: Possible alternatives for `gespenstisch`:

- schaurigen
- grusligen

Comment (respektive) [inline wiki/History_of_osu!/2022/de.md:113]: I like gruseligen tbh. Seems to be the best translation for spooky imo.
```suggestion
Für das diesjährige Halloween gab es erneut einen [Fanart-Wettbewerb](https://osu.ppy.sh/home/news/2022-10-10-halloween-fanart-contest), bei dem Künstler dazu aufgefordert wurden, die osu!-Maskottchen in gruseligen Kostümen zu zeichnen. [Dreamxiety](https://osu.ppy.sh/users/13103233) gewann den ersten Platz im Wettbewerb mit 3.749 Stimmen.[^halloween] Eine Einsendung wurde widerrufen, weil sie als zu angsteinflösend angesehen wurde und dem Autor wurde stattdessen ein eigenes Abzeichen verliehen.[^grin-reddit][^grin-twitter][^grin-badge]
```

File: wiki/History_of_osu!/2022/de.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+[BTMC](https://osu.ppy.sh/users/3171691) interviewte peppy über die Pläne für osu!(lazer). Die meisten Fragen wurden von der Community gestellt, dazwischen gab es auch einige persönliche Fragen.[^btmc-peppy-interview] Ein wichtiger Punkt war der Stand des Rankingsystems, aber es gab ebenfalls allgemeine Fragen im Bezug auf das Ökosystem von osu! und seine Zukunft.
+
+Die zwei Mods Adaptive Speed and Strict Tracking wurden in osu!(lazer) implementiert. Adaptive Speed verändert die Songgeschwindigkeit, abhängig davon, ob der Treffer früh oder spät war.[^adaptive-speed] Beim Aktivieren der Mod Strict Tracking muss sich der Cursor zu jeder Zeit im Sliderball befinden, um ein Unterbrechen der Combo zu vermeiden.[^strict-tracking]
```
Comment (respektive) [inline wiki/History_of_osu!/2022/de.md:33]:
```suggestion
Die zwei Mods Adaptive Speed und Strict Tracking wurden in osu!(lazer) implementiert. Adaptive Speed verändert die Songgeschwindigkeit, abhängig davon, ob der Treffer früh oder spät war.[^adaptive-speed] Beim Aktivieren der Mod Strict Tracking muss sich der Cursor zu jeder Zeit im Sliderball befinden, um ein Unterbrechen der Combo zu vermeiden.[^strict-tracking]
```

File: wiki/History_of_osu!/2022/de.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+
+[^aptb-results]: [Newsbeitrag von Ephemeral (08.05.2022) "A Place To Belong Fanart Contest Results"](https://osu.ppy.sh/home/news/2022-05-08-aptb-fanart-results)
+[^lazer-chat]: [Pull-Request von jai-x (23.05.2022) "Integrate new chat into game"](https://github.com/ppy/osu/pull/18377)
+[^lazer-mod-select]: [Pull-Request von bdach (05.05.2022) "Replace old mod overlay with new design"](https://github.com/ppy/osu/pull/18111)
+
+[^repel]: [Pull-Request von ggliv (07.06.2022) "Add repel mod to the osu ruleset"](https://github.com/ppy/osu/pull/18607)
+[^community-contributor]: [Newsbeitrag von osu! team (30.06.2022) "Community Contributors: 2021"](https://osu.ppy.sh/home/news/2022-06-30-community-contributors-2021)
+
+[^osu-weekly]: [Newsbeitrag von Nyquill (07.06.2017) "osu!weekly #106"](https://osu.ppy.sh/home/news/2017-06-07-osu-weekly-106)
+[^monthly-quiz]: [Newsbeitrag von Soetch, Blushing, Wowcake, 0x84f, and Hubz (01.08.2022) "osu!monthly returns, July 2022"](https://osu.ppy.sh/home/news/2022-08-01-osumonthly-1#monthly-community-quiz)
```
Comment (respektive) [inline wiki/History_of_osu!/2022/de.md:172]:
```suggestion
[^monthly-quiz]: [Newsbeitrag von Soetch, Blushing, Wowcake, 0x84f, und Hubz (01.08.2022) "osu!monthly returns, July 2022"](https://osu.ppy.sh/home/news/2022-08-01-osumonthly-1#monthly-community-quiz)
```

File: wiki/History_of_osu!/2022/de.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+[^roundtable-winners]: [Newsbeitrag von Soetch, Blushing, Wowcake, 0x84f, and Hubz (01.08.2022) "osu!monthly returns, July 2022"](https://osu.ppy.sh/home/news/2022-08-01-osumonthly-1#what's-new-in-osu!?)
```
Comment (respektive) [inline wiki/History_of_osu!/2022/de.md:173]:
```suggestion
[^roundtable-winners]: [Newsbeitrag von Soetch, Blushing, Wowcake, 0x84f, und Hubz (01.08.2022) "osu!monthly returns, July 2022"](https://osu.ppy.sh/home/news/2022-08-01-osumonthly-1#what's-new-in-osu!?)
```

File: wiki/History_of_osu!/2022/de.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+[^presets]: [Pull-Request von bdach (07.08.2022) "Add mod preset column to solo mod select overlay"](https://github.com/ppy/osu/pull/19622)
+
+[^15-anniversary-results]: [Newsbeitrag von Ephemeral (15.09.2022) "15th Anniversary Art Contest Results"](https://osu.ppy.sh/home/news/2022-09-15-anniversary-art-contest-results)
+[^15-celebration]: [YouTube-Video von 2th - osu! Archive (18.09.2022) "[Live] ppy | celebrating 15 years of osu! {09-17-2022}"](https://www.youtube.com/watch?v=xEfXbNbkIvw)
+[^argon]: [Pull-Request von peppy (19.09.2022) "Add new default "argon" skin"](https://github.com/ppy/osu/pull/20377)
+[^osu-sr-pp]: [Newsbeitrag von osu! pp committee (30.09.2022) "Changes to osu! Star Rating & Performance Points"](https://osu.ppy.sh/home/news/2022-09-30-changes-to-osu-sr-and-pp)
+[^osu-taiko-sr-pp]: [Newsbeitrag von osu!taiko pp committee (28.09.2022) "Changes to osu!taiko Star Rating & Performance Points"](https://osu.ppy.sh/home/news/2022-09-28-changes-to-osu-taiko-sr-and-pp)
+[^medals-september]: [Newsbeitrag von 0x84f, Pisapou, and RandomeLoL (02.10.2022) "osu!monthly, September 2022"](https://osu.ppy.sh/home/news/2022-10-02-osumonthly-3)
```
Comment (respektive) [inline wiki/History_of_osu!/2022/de.md:184]:
```suggestion
[^medals-september]: [Newsbeitrag von 0x84f, Pisapou, und RandomeLoL (02.10.2022) "osu!monthly, September 2022"](https://osu.ppy.sh/home/news/2022-10-02-osumonthly-3)
```

File: wiki/History_of_osu!/2022/de.md [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+[^freeze-frame]: [Pull-Request von mk56-spn (16.09.2022) "Add "Freeze Frame" mod"](https://github.com/ppy/osu/pull/20345)
+[^highest-rank]: [Pull-Request von cl8n (18.10.2022) "Show highest achieved rank on user profile"](https://github.com/ppy/osu-web/pull/9388)
+[^halloween]: [Newsbeitrag von Ephemeral (29.10.2022) "Halloween 2022 Fanart Contest Results"](https://osu.ppy.sh/home/news/2022-10-29-halloween-fanart-results)
+[^grin-reddit]: [Reddit-Kommentar von u/pepppppy in r/osugame (30.10.2022) in "osu seriously booted another submission as “too scary” again"](https://www.reddit.com/r/osugame/comments/yhh6hd/comment/iuh6ii2/)
+[^grin-twitter]: [Tweet von xx_aether (29.10.2022)](https://twitter.com/xx_aether/status/1586438659292569600)
+[^grin-badge]: [osu!-Profil von Roro-san](https://osu.ppy.sh/users/11084479)
+
+[^medals-november]: [Newsbeitrag von 0x84f, Junihuhn, Pisapou, and RandomeLoL (01.12.2022) "osu!monthly, November 2022"](https://osu.ppy.sh/home/news/2022-12-01-osumonthly-5#new-medals)
```
Comment (respektive) [inline wiki/History_of_osu!/2022/de.md:199]:
```suggestion
[^medals-november]: [Newsbeitrag von 0x84f, Junihuhn, Pisapou, und RandomeLoL (01.12.2022) "osu!monthly, November 2022"](https://osu.ppy.sh/home/news/2022-12-01-osumonthly-5#new-medals)
```

**Review by respektive — CHANGES_REQUESTED** [review]

File: wiki/History_of_osu!/2022/de.md [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+
+Das dritte osu! Community Meeting des Jahres am 6. Februar wurde zu verschiedenen Fragen aus der Community abgehalten.[^community-meeting-questions] Am 20. Februar fand das nächste Community-Meeting statt, bei dem es um Mod-Multiplikatoren und Score ging, da osu!(lazer) für einige Mods variierbare rate adjustments<!--Ratenanpassung, irgendetwas mit Inkrement? see https://docs.google.com/document/d/13cMCrQN4vvaQFA59BmFHk6D7PH9fwT2ANmhkm5cmzxE/edit --> erlaubte. Beispielsweise kann man Double Time auf 1,01-fache Geschwindigkeit einstellen, anstatt auf nur 1,5-fache wie in stable. Dadurch würden Spieler einen unfairen Vorteil auf den Ranglisten erhalten, da die Spieler trotzdem den unveränderlichen 1,12-fachen Score-Multiplikator erhielten. Infolgedessen wurden Mod-Multiplikatoren temporär deaktiviert. Das osu!-Team möchte in der Zukunft an einer robusteren Lösung arbeiten.[^community-meeting-mod]
```
[the remark this answers is not in the document]
Comment (The-Last-Cookie) [reply wiki/History_of_osu!/2022/de.md:19]: In the end, it's an adjustment in speed, so yeah seems to fit.

File: wiki/History_of_osu!/2022/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/History_of_osu!/2022/de.md:113]: I tried to find a word which carried this atmosphere from the event over as best as possible. What I mean is this feeling of drawing images that are kind of spooky and have these cutesy art styles but are obviously not Outlast-like creepy. I'm okay with your suggestion!

**Review by respektive — APPROVED** [review]

**Pull request merged**